xbmc/addons/Addon.h

```diff
@@ -128,103 +128,193 @@ class CAddon : public IAddon
     return it != m_addonInfo->Art().end() ? it->second : "";
   }
 
-  /*! \brief Check whether the this addon can be configured or not
-   \return true if the addon has settings, false otherwise
-   \sa LoadSettings, LoadUserSettings, SaveSettings, HasUserSettings, GetSetting, UpdateSetting
+  std::vector<uint32_t> GetKnownInstanceIds() const override;
+
+  /*!
+   * \brief Ask addon supports independent settings about used instances.
+   *
+   * \return true if the addon has allow instance settings, false otherwise
```

Comment:
return true if the add-on supports individual settings per add-on instance, false otherwise